client/components/signup-form/social.jsx

```diff
@@ -48,22 +46,14 @@ class SocialSignupForm extends Component {
 		const redirectUri = uxMode === 'redirect' ? `https://${ window.location.host }/start` : null;
 
 		return (
-			<Card className="signup-form__social">
```

Comment:
I think we should keep this `Card` here as otherwise it kind of breaks our guidelines regarding components (i.e. `signup-form__social` shouldn't be defined in a parent component). We could also take advantage of [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction) to distribute our blocks.

Reply:
We should simply rename our css classes in this case I think